examples/dmri_connectivity.py

```diff
@@ -26,9 +26,8 @@
 
     * http://db.tt/1vx4vLeP
 
-Along with `Camino <http://web4.cs.ucl.ac.uk/research/medic/camino/pmwiki/pmwiki.php?n=Main.HomePage>`_,
-`Camino-Trackvis <http://www.nitrc.org/projects/camino-trackvis/>`_, `FSL <http://www.fmrib.ox.ac.uk/fsl/>`_,
-and `Freesurfer <http://surfer.nmr.mgh.harvard.edu/>`_, you must also have the Connectome File Format
+Along with Camino_, Camino2Trackvis_, FSL_, and FreeSurfer_,
+you must also have the Connectome File Format
```

Comment:
Just a note that we should update https://github.com/niflows/nipype1-examples with any changes you make here, as `examples/` will be removed from nipype in a coming minor release.

Reply:
They will also require intersphinx for linking